easybuild/easyconfigs/m/matplotlib/matplotlib-1.5.1-CrayGNU-2015.11-Python-2.7.11.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '1.5.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ('python -c "import %(ext_name)s"', '')
```

Comment:
this line is not needed, it was removed in other `matplotlib` easyconfigs already, cfr. #2685